src/Plugin/VoteUpDownWidgetInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\vud\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Vote Up/Down Widget plugins.
+ */
+interface VoteUpDownWidgetInterface extends PluginInspectionInterface {
+
+  /**
+   * Method to return the label of the plugin whenever required
```

Comment:
Please see [documentation on how to add summary lines](https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#functions), one extract:

> For most functions (see exceptions below), summary lines must start with a third person singular present tense verb, and they must explain what the function does. Example: "Calculates the maximum weight for a list."

This change applies for most methods in this PR, please fix them.